<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_SignUp_Dialog</class>
 <widget class="QDialog" name="admin_SignUp_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理员注册</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mysource.qrc">
    <normaloff>:/images/物业管理.png</normaloff>:/images/物业管理.png</iconset>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Help&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;You can visit this URL to &lt;/span&gt;&lt;a href=&quot;https://www.google.com&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;www.google.com&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt; help!&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Developer: ZhangFuyun&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Collage: ShanDongJianZhu University&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;StudentId: 20140815020 &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="frame_2">
     <property name="styleSheet">
      <string notr="true">#frame_2{border:2px solid rgb(170,85,0)}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QFrame" name="frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="label">
                 <property name="minimumSize">
                  <size>
                   <width>111</width>
                   <height>51</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>111</width>
                   <height>51</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>宋体</family>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>登 陆 名：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="minimumSize">
                  <size>
                   <width>111</width>
                   <height>51</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>111</width>
                   <height>51</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>宋体</family>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>登录密码：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="minimumSize">
                  <size>
                   <width>111</width>
                   <height>51</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>111</width>
                   <height>51</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>宋体</family>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>确认密码：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="minimumSize">
                  <size>
                   <width>111</width>
                   <height>51</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>111</width>
                   <height>51</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>宋体</family>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>姓    名：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="minimumSize">
                  <size>
                   <width>111</width>
                   <height>51</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>111</width>
                   <height>51</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>宋体</family>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>联系电话：</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLineEdit" name="lineEdit_logName">
                 <property name="minimumSize">
                  <size>
                   <width>311</width>
                   <height>51</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>311</width>
                   <height>51</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>宋体</family>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="placeholderText">
                  <string>汉字、字母或数字开头</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_pwd">
                 <property name="minimumSize">
                  <size>
                   <width>311</width>
                   <height>51</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>311</width>
                   <height>51</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>宋体</family>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                 <property name="placeholderText">
                  <string>长度大于6位，建议字母数字符号混合型密码</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_pwdConfirm">
                 <property name="minimumSize">
                  <size>
                   <width>311</width>
                   <height>51</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>311</width>
                   <height>51</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>宋体</family>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                 <property name="placeholderText">
                  <string>与登录密码一致</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_realname">
                 <property name="minimumSize">
                  <size>
                   <width>311</width>
                   <height>51</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>311</width>
                   <height>51</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>宋体</family>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="placeholderText">
                  <string>请输入真实姓名</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_tel">
                 <property name="minimumSize">
                  <size>
                   <width>311</width>
                   <height>51</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>311</width>
                   <height>51</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>宋体</family>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="placeholderText">
                  <string>7位电话号码或11位手机号码</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_signup">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>51</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>51</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>提交注册信息</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder>frame_2</zorder>
  <zorder>verticalSpacer</zorder>
  <zorder>verticalSpacer_2</zorder>
  <zorder>horizontalSpacer_5</zorder>
  <zorder>horizontalSpacer_6</zorder>
 </widget>
 <resources>
  <include location="mysource.qrc"/>
 </resources>
 <connections/>
</ui>
